content/posts/2019-11-07-ln-split-tickets-decred-blog.md

```diff
@@ -0,0 +1,159 @@
+
+Over 1000 tickets have voted and over 100 are live right now, as of this writing.
+
+These are modest numbers when compared to the regular individual staking options. But they do show there is demand for the service even in the midst of a downward trend for price, both in BTC and fiat terms which allow interested parties to more easily acquire DCR and participate in the staking process as solo or VSP voters.
+
+While it's clear that it is useful, the current on-chain solution for multi-owner tickets has severe scalability issues and we've known from the start it wouldn't be around forever. It was implemented that way because that was (relatively speaking) simple and all consensus rules necessary to make it work were already in place - there was no need for any new voting agendas or code in the full node implementation.
+
+We conclude that a second layer solution is needed for split tickets. One that allows multiple owners to come together and fund tickets even if all they had was a fraction of a DCR.
+
+[Lightning Network](https://dev.lightning.community) is the current leading contestant for a permissionless second layer solution for utxo-based cryptocurrencies such as Decred and Bitcoin and the natural choice to enable large numbers of people to contribute into a single ticket.
+
+As usual, the tricky bits are the nitty gritty details of just _how_ to orchestrate purchasing the ticket and then disbursing the stake rewards after a vote such that all participants are assured their funds are recoverable in the worst case scenario of a breakdown of communication.
+
+The [on-chain version](https://matheusd.com/post/ticket-splitting-challenges/) of split tickets naturally solves this problem by relying on the consensus rules that enforce the returns of the funds of the ticket to each original financer. Off-chain multi-owner tickets however, cannot rely on that given the UTXOs are generally locked into two-party contracts. 
+
+{{< legendfigure src="/images/posts/ln-splits-01-onchainsplit.png" alt="Typical on-chain split" title="Typical on-chain split ticket with 6 participants. Notice the return addresses are guaranteed by consensus rules so funds are completely safe after a vote or revocation." >}}
+
+For example, in the Lightning Network the channel balance is defined by the UTXO which locks funds in the multisig 2-of-2 script of the channel's parties. While it would be possible to use that UTXO to directly fund a ticket, that's still an on-chain operation and ends up having the same scalability problems as the current on-chain solution. Besides that, users might not want to commit all their available channel funds into tickets or only one of the parties might be interested in doing so.
```

Comment:
```suggestion
For example, in the Lightning Network the channel balance is defined by the UTXO which locks funds in the multisig 2-of-2 script of the channel's parties. While it would be possible to use that UTXO to directly fund a ticket, that's still an on-chain operation and ends up having the same scalability problems as the current on-chain solution. Besides that, users might not want to commit all their available channel funds into tickets, or only one of the parties might be interested in doing so.
```